modules/labs-react/text-input/stories/examples/LabelPosition.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {TextInput} from '@workday/canvas-kit-labs-react/text-input';
+import {HStack} from '@workday/canvas-kit-labs-react/layout';
+
+export const LabelPosition = () => {
+  const [value, setValue] = React.useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value);
+  };
+
+  return (
+    <HStack spacing="l" alignItems="center">
```

Comment:
perhaps this could be a hook to use with `Stack`?
e.g.
`useLeftLabel = () => ({ flexDirection="row", spacing="l", alignItems="center" })`
`useTopLabel = () => ({ flexDirection="column", spacing="xxxs" alignItems="flex-start" })`

Reply:
Hmm. Or maybe a layout component? Like `HorizontalField` and `VerticalField`? If the worry is remembering what styles to set on every field.

Reply:
¯\\_(ツ)_/¯ Eh, I don't feel strongly either way so would tend towards leaving up to the consumer.